birdhouse/env.local.example

```diff
@@ -347,7 +346,6 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 #
 ### public-share paths
 #
-## /data/jupyterhub_user_data/public-share/{username}-public
```

Comment:
Will the shared/public aspect presented in https://github.com/bird-house/birdhouse-deploy/pull/298#issuecomment-1452189327 be defined in a subsequent PR? It looks like these variable might need conditional activation/update based on `cowbird` being enabled or not.

Reply:
Yes, I think this path could change when enabling `cowbird`. I will revert this change (and the other comment example above) for now, since this PR doesn't include changes for the shared/public aspects.

Let me know if you'd prefer me to implement and add the shared/public aspects directly in this PR.

Reply:
A following PR is ok.